src/components/modules/DetailedPage/ProductHeader/StyledProductHeader.js

```diff
@@ -10,17 +10,47 @@ export const Tags = styled.div`
   border-radius: 20px;
 `;
 
+export const ColumnTitle = styled.div`
+  margin-top: 1rem;
+  font-family: Abel;
```

Comment:
Please use the theme for font-family, 

example:
 `font-family: ${({ theme }) => theme.fonts.headline};` for Abel (headline)
 `font-family: ${({ theme }) => theme.fonts.normal};` for Nunito Sans (normal)